core/tools/blueos_startup_update/blueos_startup_update

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import appdirs
+import copy
+import os
+import json
+import re
+import sys
+import time
+
+DELTA_JSON = {
+    'core': {
+        'binds': {
+            '/run/udev': {
+                'bind': '/run/udev',
+                'mode': 'ro'
+            }
+        }
+    }
+}
+
+import collections
+
+# Copyright 2016-2022 Paul Durivage
+# Licensed under the Apache License, Version 2.0 (the "License");
+# Based on: https://gist.github.com/angstwad/bf22d1822c38a92ec0a9
+def dict_merge(dct, merge_dct):
+    for k, v in merge_dct.items():
+        if (k in dct and isinstance(dct[k], dict) and isinstance(merge_dct[k], dict)):  #noqa
+            dict_merge(dct[k], merge_dct[k])
+        else:
+            dct[k] = merge_dct[k]
+
+def main():
+    current_git_version = os.getenv('GIT_DESCRIBE_TAGS')
+    match = re.match(r'(?P<tag>.*)-(?P<commit_number>\d+)-(?P<commit_hash>[a-z0-9]+)', current_git_version)
+    tag, commit_number, commit_hash = match['tag'], match['commit_number'], match['commit_hash']
+    print(f"Running BlueOS: {tag=}, {commit_number=}, {commit_hash=}")
```

Comment:
was this data supposed to be consumed somwhere?

Reply:
No, the idea is to make it available if version check is necessary inside BlueOS, as possible necessary in this script. Right now there is no way to know BlueOS version inside core.